dnf/cli/commands/module.py

```diff
@@ -299,7 +299,7 @@ def run_on_module(self):
     class ProvidesSubCommand(SubCommand):
 
         aliases = ("provides", )
-        summary = _('list modular packages')
+        summary = _('locate a module the modular packages belongs to')
```

Comment:
I think there is a typo:
`locate a module the modular packages belong to` (belongs -> belong)